### DISCUSSION TITLE
Auto enable or disable HDR on startup

### DISCUSSION THREAD

**Discussion by flowerdealer** [Codectory/AutoActions#22] [Ideas]
Just because the profiles could potentially leave you with HDR on (or off) when you restart your machine, it would be nice if we had an option to toggle hdr on or off on startup to ensure our preferred setting. The option to have it on makes more sense in the context of my request that the program have an option to disable HDR on a profile activation, instead of just enabling.

**Comment by Codectory** [comment]
This should be possible. App based on/off with a priority.
I became a father two weeks ago, see you need a bit  patience for this ;)

flowerdealer ***@***.***> schrieb am Do. 15. Apr. 2021 um
20:15:

**Reply by flowerdealer** [reply]
Congratulations! Also a parent myself so I understand. I already have my modified version that has the behavior that I want (profile disabling instead of enabling and enabling on autostart), but I think this will open up your software to a new group of people like me (VR desktop users and people that remote into their desktop).  Great work!

As a side note, and forgive my ignorance, what type of programming project is this? I'm just curious as I would like to learn a bit more. I was able to build and modify it without much issue but other than that I don't understand much about how it's structured, what frameworks it's using, etc., and would love to learn more.

**Comment by Codectory** [comment]
It's a C# project, a WPF project in Visual Studio, so I am using the .Net Framework and MVVM as design pattern, although I am quite sure, that I broke up with the pattern at some points. 

Some basic stuff (Windows, base classes for MVVM) are referenced by some libraries I created for my self in the past. For other parts like the UWP integration or packaging all binaries in one executable, I use some Nuget packages. The "magic" happens in a referenced C++ project (HDRController). The code was initially copied from another Git repo, as I am not very familiar with C++, but I managed to change the code for my needs, but don't use it as a good example. It's just working ;)
So yeah, that's all + Google.

**Comment by flowerdealer** [comment]
Thanks for the info, and good luck with parenthood!

**Comment by flowerdealer** [comment]
Hey, just checking back on this request. Is there any way to default to HDR on for windows on startup? Thanks for the great work.

**Comment by Codectory** [comment]
Not possible, but a workaround would be to add for example explorer.eye to
AutoActions and assign a profile which activates hdr.

flowerdealer ***@***.***> schrieb am Fr. 14. Jan. 2022 um
18:41:
